zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -182,6 +182,10 @@ private static DecisionInstanceFilter toDecisionInstanceFilter(
           .map(s -> convertEnum(s, DecisionInstanceState.class))
           .ifPresent(builder::states);
       ofNullable(filter.getEvaluationFailure()).ifPresent(builder::evaluationFailures);
+      ofNullable(filter.getEvaluationDate())
+          .map(SearchQueryRequestMapper::toOffsetDateTime)
+          .map(f -> List.of(Operation.gte(f), Operation.lt(f)))
+          .ifPresent(builder::evaluationDateOperations);
```

Comment:
🔧 Let's maybe treat this as an equals for now? Is that even possible? 🤔

Reply:
It should be possible, I'm changing it. This field is currently not tested and will change with next iteration, I'll also add tests there.